addons/crm/tests/test_crm_pls.py

```diff
@@ -359,6 +395,7 @@ def test_predictive_lead_scoring(self):
 
         # Continue to test probability computation
         leads[3].probability = 40
+        leads[3].invalidate_recordset()
```

Comment:
Should we test the edge cases as well?

Like for example writing 50 probability on a lead that is in "stage won" (the famous "not supposed to happen").